benchmarks/bench_ioc.py

```diff
@@ -0,0 +1,59 @@
+import os
+from . import util
+
+
+class IocHandler:
+    def __init__(self):
+        # NOTE: have to increase EPICS_CA_MAX_ARRAY_BYTES if NELM >= 4096
+        #       (remember default is 16384 bytes / sizeof(int32) = 4096)
+        self._cms = []
+        self._softioc_processes = []
+
+    def setup_ioc(self, *, db_text, max_array_bytes=16384, env_vars=None,
+                  **kwargs):
+        if env_vars is None:
+            env_vars = {}
+
+        env = dict(EPICS_CA_MAX_ARRAY_BYTES=str(max_array_bytes))
+        os.environ['EPICS_CA_MAX_ARRAY_BYTES'] = str(max_array_bytes)
```

Comment:
`os.environ` doesn't get passed to the subprocess, so that's why `env` is here. But good point, this line isn't really necessary.